e2e_testing/torchscript/elementwise.py

```diff
@@ -481,6 +481,43 @@ def ElementwiseRsqrtModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
 
 # ==============================================================================
+
+class ElementwiseAbsModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+
+    def forward(self, a):
+        return torch.abs(a)
+
+@register_test_case(module_factory=lambda: ElementwiseAbsModule())
+def ElementwiseAbsModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5).uniform_(-1.0, 0.0))
```

Comment:
The tu.rand api accept range argument and you can use the arguments to specify range instead of using `uniform_` here. See https://github.com/llvm/mlir-npcomp/blob/56c6e3676bbe1571dab0c7cc1fb5080846819438/python/torch_mlir_e2e_test/torchscript/framework.py#L151-L152

Reply:
ok, yeah, I got it! Thanks!